spec/commands/deploy.spec.ts

```diff
@@ -330,6 +330,226 @@ describe("deploy", () => {
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "BUILDING_ASSETS",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "UPLOADING_ASSETS",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "CONVERGING_STORAGE",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "PUBLISHING_TREE",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "RELOADING_SANDBOX",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      await publishStatus.emitResponse({
+        data: {
+          publishStatus: {
+            publishStarted: true,
+            remoteFilesVersion: "1",
+            progress: "COMPLETED",
+            issues: [],
+            status: {
+              code: "Pending",
+              message: undefined,
+              output: "https://test.gadget.app/url/to/logs/with/traceId",
+            },
+          },
+        },
+      });
+
+      expectStdout().toMatchInlineSnapshot(`
+        "Deploying development to test.gadget.app (​https://test.gadget.app/​)
+
+        ⠙ Calculating file changes.
+        ✔ Your files are up to date. 12:00:00 AM
+
+        ! Problems found
```

Comment:
Should we be printing `! Problems found` when there are only `deletedModelsAndFields` in the response but no `issues`?

Reply:
Woops good catch, I gotta update the text in general to match more with:

![CleanShot 2024-08-28 at 16 48 39@2x](https://github.com/user-attachments/assets/17b321ac-dca6-42e3-a7b7-a4f6e6317f7f)